### DISCUSSION TITLE
iTwin Synchronizer save to local server

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1537] [Q&A]
is  iTwin Synchronizer can save or link local LAN  server ？？

**Comment by pmconne** [comment]
@jffmarker

**Reply by jffmarker** [reply]
At this time, the iTwin Synchronizer only works with the iModel Hub. Are you trying to synchronize changes to a hosted iTwin Stack instance? We are considering finding ways to allow the iTwin Synchronizer to do this, but current synchronization to the iModel Bank (part of the iTwin Stack) must be done manually.
